app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,8 @@ html lang="#{I18n.locale}"
       = content_for?(:title) ? APP_NAME + ' - ' + yield(:title) : APP_NAME
 
     == stylesheet_link_tag 'application', media: 'all'
-    == javascript_include_tag 'application'
+    - unless controller_name == 'home'
+      == javascript_include_tag 'application'
```

Comment:
optimization -- the homepage doesn't require js, so for performance sake, i'd prefer to not load it; if you know of a better way to do this, i'm all 👂 s :)